src/tests/functional/plugin/shared/src/behavior/compiled_model/import_export.cpp

```diff
@@ -298,6 +298,20 @@ TEST_P(OVClassCompiledModelImportExportTestP, smoke_ImportNetworkNoThrowWithDevi
     OV_ASSERT_NO_THROW(executableNetwork.create_infer_request());
 }
 
+TEST_P(OVClassCompiledModelImportExportTestP, smoke_ImportNetworkThrowWithDeviceName) {
+    SKIP_IF_CURRENT_TEST_IS_DISABLED();
+    ov::Core ie = createCoreWithTemplate();
+    std::stringstream xmlFile;
+    std::stringstream binFile;
+    // Import model with IR input throw exception
+    ov::pass::Manager manager;
+    manager.register_pass<ov::pass::Serialize>(xmlFile, binFile);
+    manager.run_passes(actualNetwork);
+    OV_EXPECT_THROW((ie.import_model(xmlFile, target_device)),
+                    ov::Exception,
```

Comment:
I don't see removing of `bin` and `xml`

Reply:
Hi, `bin` and `xml` here are not saved to the disk, they are serialized to string buffer in memory. The test here is to simulate that the `import_model` accepts wrong input which is a pure IR file rather than the stream exported from `export_model`. This situation is the described by the issue reported by [CVS-126392](https://jira.devtools.intel.com/browse/CVS-126392)

Reply:
remove `xml` and `bin` in 10eeb73c23a9f0debe70b847f55ac910ff153ec2